PKGBUILD

```diff
@@ -1,46 +1,74 @@
 # Maintainer: Reto Brunner <brunnre8@gmail.com>
 # Maintainer: Maxime Poulin <maxpoulin64@gmail.com>
 pkgname=thelounge
-pkgver=4.3.0
-pkgrel=2
+pkgver=4.3.1
+pkgsuffix=-rc.1
+pkgrel=4
```

Comment:
You can reset pkgrel back to 1 whenever pkgver increases, pkgrel is for re-releases of the same package with packaging changes.

Reply:
I know, guess how many iterations I had to do? 😉 

We anyhow need to adapt it to remove the prerelease string and add the .SRCINFO prior to publishing it